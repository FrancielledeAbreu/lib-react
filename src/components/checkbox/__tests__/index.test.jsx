import React from 'react'
import Checkbox from '../index'
import renderer from 'react-test-renderer'

describe('Snapshot', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Checkbox inputCheck={['String-one', 'String-two']} children='Hello' />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })

  it('renders empty string', () => {
    const tree = renderer
      .create(
        <Checkbox inputCheck={['', '', '']} children='Hello Lib-Kenzie' />
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
