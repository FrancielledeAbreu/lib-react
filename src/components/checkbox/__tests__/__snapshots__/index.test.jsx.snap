// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders correctly 1`] = `
<div
  className="sc-bdnylx kaUBSj"
>
  <h2>
    Hello
  </h2>
  <label
    className="main"
  >
    String-one
    <input
      type="checkbox"
    />
    <span
      className="mark"
    />
  </label>
  <label
    className="main"
  >
    String-two
    <input
      type="checkbox"
    />
    <span
      className="mark"
    />
  </label>
</div>
`;

exports[`Snapshot renders empty string 1`] = `
<div
  className="sc-bdnylx kaUBSj"
>
  <h2>
    Hello Lib-Kenzie
  </h2>
  <label
    className="main"
  >
    
    <input
      type="checkbox"
    />
    <span
      className="mark"
    />
  </label>
  <label
    className="main"
  >
    
    <input
      type="checkbox"
    />
    <span
      className="mark"
    />
  </label>
  <label
    className="main"
  >
    
    <input
      type="checkbox"
    />
    <span
      className="mark"
    />
  </label>
</div>
`;
